demo/demo-springmvc/springmvc-client/src/main/java/io/servicecomb/demo/springmvc/client/SpringmvcClient.java

```diff
@@ -97,24 +97,22 @@ public static void run() throws Exception {
 
       TestMgr.check(String.valueOf(true), String.valueOf(resultMap.get("CPU and Memory").contains("heapUsed=")));
 
-      TestMgr.check(resultMap.get("totalRequestProvider OPERATIONAL_LEVEL"),
-          "{springmvc.codeFirst.saySomething=3, springmvc.codeFirst.testRawJsonAnnotation=3, " +
-              "springmvc.codeFirst.sayHi2=3, springmvc.codeFirst.responseEntity=6, springmvc.codeFirst.fileUpload=3, " +
-              "springmvc.codeFirst.responseEntityPATCH=3, springmvc.codeFirst.textPlain=3, " +
-              "springmvc.codeFirst.metricsForTest=1, springmvc.codeFirst.testform=6, " +
-              "springmvc.controller.saySomething=6, springmvc.codeFirst.fallbackReturnNull=6, " +
-              "springmvc.codeFirst.addString=3, springmvc.codeFirst.reduce=3, springmvc.codeFirst.sayHi=3, " +
-              "springmvc.codeFirst.cseResponse=6, springmvc.codeFirst.bytes=3, springmvc.controller.sayHei=3, " +
-              "springmvc.codeFirst.fallbackThrowException=9, springmvc.codeFirst.testModelWithIgnoreField=1, " +
-              "springmvc.codeFirst.testUserMap=3, springmvc.codeFirst.isTrue=3, springmvc.codeFirst.add=3, " +
-              "springmvc.codeFirst.fallbackFromCache=6, springmvc.controller.sayHi=17, springmvc.codeFirst.sayHello=6,"
-              +
-              " springmvc.controller.sayHello=6, springmvc.codeFirst.addDate=3}");
-
-      TestMgr.check(String.valueOf(resultMap.get("RequestQueueRelated").contains("springmvc.codeFirst.saySomething")),
-          String.valueOf(true));
-      TestMgr.check(String.valueOf(resultMap.get("RequestQueueRelated").contains("springmvc.controller.sayHi")),
-          String.valueOf(true));
+      String[] requestProviders = resultMap.get("totalRequestProvider OPERATIONAL_LEVEL").replace("{", "")
```

Comment:
no need to convert everything to string to compare

Reply:
DONE